2996_reflection/reflection.md

```diff
@@ -3122,6 +3156,31 @@ has its linkage determined as follows:
 
 :::
 
+Add a bullet to paragraph 13:
+
+::: std
+
+[13]{.pnum} A declaration `$D$` _names_ an entity `$E$` if
+
+* `$D$` contains a _lambda-expression_ whose closure type is `$E$`,
+* [`$D$` contains a _splice-specifier_ designating `$E$`,]{.addu}
+
+...
```

Comment:
When adding bullets, always number them. And them might as well provide the other two bullets for context.

```suggestion
* [13.1]{.pnum} `$D$` contains a _lambda-expression_ whose closure type is `$E$`,
* [13.1+]{.pnum} [`$D$` contains a _splice-specifier_ designating `$E$`,]{.addu}
* [13.2]{.pnum} `$E$` is not a function or function template and `$D$` contains an *id-expression*, *type-specifier*, *nested-name-specifier*, *template-name*, or *concept-name denoting* `$E$`, or
* [13.#]{.pnum} `$E$` is a function or function template and `$D$` contains an expression that names `$E$` ([basic.def.odr]) or an *id-expression* that refers to a set of overloads that contains `$E$`.
```

Reply:
Done, thanks.